Wireframe/style.css

```diff
@@ -49,11 +66,14 @@ main {
   max-width: var(--container);
   margin: 0 auto calc(var(--space) * 4) auto;
 }
-footer {
-  position: fixed;
+footer{
   bottom: 0;
   text-align: center;
+  width: 100%;
+  display: flex;
+  justify-content: center;
```

Comment:
There is an acceptance criterion for this exercise:

- The page footer is fixed to the bottom of the viewport.

What might we use here to make sure this criterion is met?